rmgpy/statmech/conformerTest.py

```diff
@@ -32,6 +32,8 @@
 This script contains unit tests of the :mod:`rmgpy.statmech.conformer` module.
 """
 
+from __future__ import division
```

Comment:
Do we need to keep this statement? Wasn't it added just for `futerize` to function as we intended? @mliu49

Reply:
Correct, we do not need this statement and it should be removed.

Reply:
I've been keeping the division imports so that we can confirm that we won't break anything by using float division while we're testing in Python 2.

Reply:
You said though that this was fine to remove if we have inspected all of the division cases and also ran the tests to confirm, right?